app/modules/teams/resources.py

```diff
@@ -98,11 +99,8 @@ def patch(self, args, team):
                 db.session,
                 default_error_message="Failed to update team details."
             ):
-            for operation in args:
-                if not self._process_patch_operation(operation, team=team):
-                    log.info("Team patching has ignored unknown operation %s", operation)
-
-            db.session.merge(team)
+            if parameters.PatchTeamDetailsParameters.perform_patch(args, obj=team):
```

Comment:
I think the function will raise an exception, so there is no need to do `if` anymore.